modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/renderer/ObjectEntryRowInfoItemRenderer.java

```diff
@@ -190,7 +212,39 @@ else if (Objects.equals(
 							return StringPool.BLANK;
 						}
 
-						return dlFileEntry.getFileName();
+						Link fileEntryLink = new Link() {
+							{
+								href = StringBundler.concat(
+									_portal.getPathContext(),
+									_portal.getPathMain(), "/portal/login");
+								label = dlFileEntry.getFileName();
+							}
+						};
+
+						try {
+							FileEntry fileEntry = _dlAppService.getFileEntry(
```

Comment:
Hey @carolmariaabb , I don't know what's the difference between DLFileEntry and FileEntry, but I think we can get rid of the use of method fetchDLFileEntry invoked before and just use FileEntry to get the file name like I saw [here](https://github.com/liferay/liferay-portal/blob/b38777bdf00d43dcb9d978655717a3c5886179e1/modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/java/com/liferay/object/dynamic/data/mapping/form/field/type/internal/attachment/AttachmentDDMFormFieldTemplateContextContributor.java#L151). Is that possible?

Reply:
I found this [documentation](https://help.liferay.com/hc/en-us/articles/360029045591-Getting-Files) and after reading it, it seemed to me that the correct entity to use is FileEntry. Let me know if you have more context about it.

Reply:
I created this [Technical Debt](https://issues.liferay.com/browse/LPS-175512) to address this when possible